.github/workflows/How_To_Release.md

```diff
@@ -0,0 +1,48 @@
+# How to publish a new HyTEST release
+
+## Essential steps
+* Make `dev` branch look like you want.  
+  * Merge various feature branches in to `dev` with PRs (or not... manually also works; PR preferred)
```

Comment:
did we decide to use a formal dev branch? Sorry I've been a little out of the loop the past few weeks

Reply:
I don't know if we did or not... this system will work either way... but it will fire with every merge into main. So if we merge into main a lot, then it will make a new release each time. 

I can also make this an action triggered by different events -- or even manually.